libr/util/str.c

```diff
@@ -3564,6 +3565,20 @@ R_API const char *r_str_str_xy(const char *s, const char *word, const char *prev
 	return d;
 }
 
+R_API size_t r_charset_encode_str (ut8 *asciistr, ut8 *in, size_t len_input, charset *custom_charset, size_t len_charset) {
+	size_t i;
+	size_t j;
+	for(i = 0; i < len_input; i ++) {
+		for(j = 0; j < len_charset; j ++) {
+			if (in[i] == custom_charset[j].hx) {
```

Comment:
thats innefficient. a hashtable can do that in O(1)

Reply:
@trufae  I want to allow the user to make a plugin with less than 255 chars.

Reply:
if it's 256, just have a LUT?